main/lsp/requests/rename.cc

```diff
@@ -44,12 +44,24 @@ bool isSubclass(const core::GlobalState &gs, core::SymbolRef root, core::SymbolR
     }
     visited[s.classOrModuleIndex()] = true;
 
-    auto super = s.data(gs)->superClass();
```

Comment:
This whole function is named `isSubclass`, but that is no longer what you are checking for. Now you are checking if a class `derivesFrom` another class? We should rename it.